test/test_autograd.py

```diff
@@ -6152,6 +6152,18 @@ def test_inplace_view_non_contig(self, device):
         x.sum().backward()
         self.assertEqual(root.grad.tolist(), [[1, 2], [1, 1], [1, 1]])
 
+    def test_mv_grad_stride_0(self, device):
+        # Reference: https://github.com/pytorch/pytorch/issues/38315
+        mat = torch.randn(2, 2, device=device)
```

Comment:
You will need to change these to `dtype=torch.double` for the gradcheck to be happy.
It is because all the default threshold are set assuming double precision.
This should fix the CI :)